masonite/request.py

```diff
@@ -143,12 +143,14 @@ def _get_standardized_value(self, value):
     def app(self):
         return self.container
 
-    def has(self, name):
+    def has(self, name, *args):
         """
         Check if a request variable exists
         """
-
-        return name in self.request_variables
+        if args:
+            return name in self.request_variables and all((arg in self.request_variables) for arg in args)
+        else:
```

Comment:
So i think there is a bit of redundant coding here. What we should be able to do is have the method signature like:

```python
def has(self, *args):
```

And we don't need the name at all. Whether we pass 1 parameter or 50 it should still work. It would just be a list of 1 element.